commands.py

```diff
@@ -589,10 +561,17 @@ async def poll(self, ctx):
                         if reactor.id != self.bot.user.id:
                             tally[reaction.emoji] += 1
 
-            output = f"{'Current' if question == 'check' else 'Final'} results of the poll **\"{embed.title}\"**\nLink: {poll_message.jump_url}\n```"
+            output = f"{'Final' if end else 'Current'} results of the poll **\"{embed.title}\"**\nLink: {poll_message.jump_url}\n```"
 
-            for key in tally.keys():
-                output += f"{options_dict[key]}: {'▓' * tally[key] if tally[key] == max(tally.values()) else '░' * tally[key]} ({tally[key]} votes, {round(tally[key] / sum(tally.values()) * 100 if sum(tally.values()) else 0, 2)}%)\n\n"
+            max_len = max(map(len, tally.values()))
+            # Dicts act like sets of their keys without .values() or something similar. So .keys() is rarely needed.
```

Comment:
lmao some of these things I just don't remember to fix

this code was written like a year and a half ago when I didn't know as much of the ins and outs of Python